### DISCUSSION TITLE
Presence = ANY presence

### DISCUSSION THREAD

**Discussion by jasonjordan** [ESPresense/ESPresense#485] [Q&A]
Hi,

I have a shared outdoor space. I could use a PIR to control the lights, but if people are sitting still, the lights could go out.

ESPresence seems like the perfect solution to avoid that - but I'd like to be able to have the lights turned on by the presence of ANY ble device - not just a defined one.  And of course, once NO ble device is detected, turn the lights off.

Is there a way to achieve this?

Cheers, Jas

**Comment by DTTerastar** [comment]
count can take an * i believe.  Should work to count any ble device.

**Reply by gnome4711** [reply]
Hi [jasonjordan](https://github.com/jasonjordan),

I solved a similar problem with my garage door. It opens when at least one of some certain beacon are near. It's  also programmed to close when when there are none of the beacons are near. In my case I use Espresense only as a "bubble-maker" via mqtt. Inside the house I keep the settings at default. That works great for that. At the garage the settings are set for sending every second a bubble for every beacon near. 

In Node Red I use a timeout-node node that starts a countdown when none of the beacons are recognized anymore. Is a beacon back while countdown is running it restarts (the countdown) by any new advertise. When countdown reaches zero the garage closes.
Of course it needs some programming no matter what system you use. But that's a way to do with your lights.
